pallets/randomness/src/lib.rs

```diff
@@ -163,54 +197,48 @@ pub mod pallet {
 	/// Number of randomness requests made so far, used to generate the next request's uid
 	pub type RequestCount<T: Config> = StorageValue<_, RequestId, ValueQuery>;
 
-	/// Current local per-block VRF randomness
-	/// Set in `on_initialize`
 	#[pallet::storage]
 	#[pallet::getter(fn local_vrf_output)]
+	/// Current local per-block VRF randomness
+	/// Set in `on_initialize`
 	pub type LocalVrfOutput<T: Config> = StorageValue<_, Option<T::Hash>, ValueQuery>;
 
-	/// Relay epoch
 	#[pallet::storage]
 	#[pallet::getter(fn relay_epoch)]
+	/// Relay epoch
```

Comment:
Can we have the docs above the derive macros as is usually done in the substrate codebase as well https://github.com/paritytech/substrate/blob/master/frame/babe/src/lib.rs#L429 which looks more idiomatic.